daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/VariableMap1.scala

```diff
@@ -455,14 +455,14 @@ class VariableMap private (
          * the way we evaluate escapeSchemes it could lead us to setting the
          * variable read too early */
         pstate.SDW(
-          WarnID.VariableSet,
+          WarnID.VariableRead,
           "Cannot set variable %s after reading the default value. State was: %s. Existing value: %s",
           variable.rd.globalQName,
-          VariableSet,
+          VariableRead,
```

Comment:
Is line 461the only line that needs to be changed?

This warning is about setting a variable, so I think the WarnID should probably still be `VariableSet`.

And the variable state needs to be set to "VariableSet" below since we are changing the variable state from Read to Set. I think all that needs to change is the warning from "State was VariableSet" to "State was VariableRead", which this line does.

Also, please rebase/squash your changes into a single commit and include the bug number in the commit message. The [Code Contributor Workflow](https://cwiki.apache.org/confluence/display/DAFFODIL/Code+Contributor+Workflow) has commands and an example commit message if you're unsure.